doc/source/command_syntax_usage.rst

```diff
@@ -141,6 +141,7 @@ this contains a large amount of information about the operating system, hardware
 - ``jboss.brms.drools-core-ver`` - Drools version
 - ``jboss.brms.kie-api-ver`` - KIE API version
 - ``jboss.brms.kie-war-ver`` - KIE runtime version
+- ``jboss.eap.common-directories`` - Presence of common directories for JBoss
```

Comment:
Perhaps in all of these we should be more explicit and say JBoss EAP

Reply:
Do you mean say EAP in the BRMS and FUSE facts, or in the documentation for the EAP facts? I'll go ahead and push a change for the second interpretation, can always do the first one too.

Reply:
in lines 144-147  `jboss.eap.common-directories`, `jboss.eap.deploy-dates`, `jboss.eap.installed-versions` descriptions all just say "JBoss" ...

Reply:
Perfect, that's what I changed.